src/nls/hr/strings.js

```diff
@@ -62,7 +62,7 @@ define({
     "ERROR_CREATING_FILE"               : "Došlo je do greške prilikom pokušaja da se kreira {0} <span class='dialog-filename'>{1}</span>. {2}",
 
     // Application preferences corrupt error strings
-    "ERROR_PREFS_CORRUPT_TITLE"         : "Greška kod čitanja preferencija",
+    "ERROR_PREFS_CORRUPT_TITLE"         : "Greška prilikom učitanja preferencija",
```

Comment:
how did u edit this in the first place? didn't  you clone the repo? if you did, you should have that branch. that is your working branch. I hope u didn't  delete it before merging.